coderd/rbac/authz_internal_test.go

```diff
@@ -666,23 +737,23 @@ func testAuthorize(t *testing.T, name string, subject subject, sets ...[]authTes
 						assert.Error(t, authError, "expected unauthorized")
 					}
 
-					partialAuthz, err := authorizer.Prepare(ctx, subject.UserID, subject.Roles, a, c.resource.Type)
+					partialAuthz, err := authorizer.Prepare(ctx, subject.UserID, subject.Roles, scope, a, c.resource.Type)
 					require.NoError(t, err, "make prepared authorizer")
 
 					// Also check the rego policy can form a valid partial query result.
 					// This ensures we can convert the queries into SQL WHERE clauses in the future.
 					// If this function returns 'Support' sections, then we cannot convert the query into SQL.
-					if len(partialAuthz.partialQueries.Support) > 0 {
-						d, _ := json.Marshal(partialAuthz.input)
+					if len(partialAuthz.mainAuthorizer.partialQueries.Support) > 0 {
+						d, _ := json.Marshal(partialAuthz.mainAuthorizer.input)
 						t.Logf("input: %s", string(d))
-						for _, q := range partialAuthz.partialQueries.Queries {
+						for _, q := range partialAuthz.mainAuthorizer.partialQueries.Queries {
 							t.Logf("query: %+v", q.String())
 						}
-						for _, s := range partialAuthz.partialQueries.Support {
+						for _, s := range partialAuthz.mainAuthorizer.partialQueries.Support {
 							t.Logf("support: %+v", s.String())
 						}
 					}
-					require.Equal(t, 0, len(partialAuthz.partialQueries.Support), "expected 0 support rules")
+					require.Equal(t, 0, len(partialAuthz.mainAuthorizer.partialQueries.Support), "expected 0 support rules")
 
```

Comment:
Technically we want to check for both prepared structures. This just ensures the rego can be partially executed. If these checks fails, our rego queries will fail on Filters.